core/src/stack/mod.rs

```diff
@@ -0,0 +1,34 @@
+use super::{range, Range};
+
+// CONSTANTS
+// ================================================================================================
+
+/// Length of a stack word.
+pub const NUM_OP_BITS: usize = 7;
```

Comment:
I think the comment is wrong here. Also, we already have a constant like this declared in `core/src/decoder/mod.rs`.

Reply:
Makes sense. I've removed it in the latest PR.